### ISSUE TITLE
Crash with unsupported media type

### ISSUE THREAD

**Issue by Malax** [basti1302/traverson#82]
I just wanted to try traverson for the first time and threw a quick script together. However, it crashed with a cryptic error message without even calling the callback passed to`getResource`.

> TypeError: Cannot read property 'step' of undefined

This is the code I tried to run with node `7.0.0` and traverson `5.0.0`:
```javascript
const traverson = require('traverson');
traverson.from('https://api.github.com/emojis').getResource((a, b) => console.log(a, b))
```
I tinkered a bit and finally found the issue (by enabling minilog). GitHub returned a `403` with content type `text/html` which seems to cause traverson to fall flat on it's nose without proper error handling.

This issue is probably related to #52.

**Comment by basti1302** [comment]
Thanks a lot for taking the time to report it. I'll look into it soon-ish.

**Comment by Malax** [comment]
Thanks for taking the time to fix this! 👍

**Comment by basti1302** [comment]
The fix has landed in 6.0.0
